src/components/Header/Header.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../themes/oneui/oneui.scss';
+
+.Header {
```

Comment:
The header is currently not using `font-family-primary`, but I suppose this should ideally be implemented globally instead of for every component

Reply:
I agree. it would be annoying to set it everywhere all the time.